test/wasm-spec/core/bulk.wast

```diff
@@ -219,7 +219,7 @@
 (assert_trap (invoke "init" (i32.const 2) (i32.const 0) (i32.const 2))
     "out of bounds table access")
 (assert_trap (invoke "call" (i32.const 2))
-    "uninitialized element")
+    "uninitialized element 2")
```

Comment:
This is a change which is wrong in the test suite. Please check 4055596080c785e92736451661a80cca02b0924c

Reply:
Fix it. Thank you for the review!